conf/pipeline/rnaseq/azurebatch_pools_Edv4.config

```
@@ -0,0 +1,61 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    nf-core/rnaseq Nextflow config file for Azure Batch pools
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines several Azure Batch pools with virtual machines from Edv4-series, 
+    assigns pools to labels according to the requirements defined in base.config.
+    You might need to adjust vmCount and maxVmCount depending on your Batch account quotas.
+
+    Use as follows:
+        nextflow run nf-core/rnaseq -c '<Path to this configuration file>'  -profile azurebatch
+            --input 'az://<Path to input file>' --outdir 'az://<Output directory>' 
+            -w 'az://<Work directory>' [<Other pipeline parameters>]
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_description = 'Azure Batch Edv4 pools profile'
+    config_profile_contact = 'Olesya Melnichenko (@olesya13) & Venkat Malladi (@vsmalladi)'
+    config_profile_url = 'https://azure.microsoft.com/services/batch/'
+}
+
+process {
+    queue = 'Standard_E2d_v4'
+    withLabel:process_low   {queue = 'Standard_E2d_v4'}
```

```suggestion
    withLabel:process_low {queue = 'Standard_E2d_v4'}
```
Might pick a lane on the alignment or no alignment here.

---

fixed - removed extra spaces